### PR TITLE
[LP-10] Fix theory deletion issue

### PR DESCRIPTION
**Pull request by IvanYarochkin**

### CODE REVIEW

File: src/main/webapp/app/entities/theory/theory-delete-dialog.component.html [hunk 1]
```diff
@@ -6,14 +6,14 @@ <h4 class="modal-title" jhiTranslate="entity.delete.title">Confirm delete operat
     </div>
     <div class="modal-body">
         <jhi-alert-error></jhi-alert-error>
-        <p id="jhi-delete-theory-heading" jhiTranslate="learnportalApp.theory.delete.question" [translateValues]="{id: theory.id}">Are you sure you want to delete this Theory?</p>
+        <p id="jhi-delete-theory-heading" jhiTranslate="learnportalApp.theory.delete.question" [translateValues]="{title: theory.title}"></p>
     </div>
     <div class="modal-footer">
         <button type="button" class="btn btn-secondary" data-dismiss="modal" (click)="clear()">
-            <fa-icon [icon]="'ban'"></fa-icon>&nbsp;<span jhiTranslate="entity.action.cancel">Cancel</span>
```
Comment (xxMikhailx) [inline src/main/webapp/app/entities/theory/theory-delete-dialog.component.html:13]: Don't delete English words in tags

File: src/main/webapp/app/entities/theory/theory-delete-dialog.component.html [hunk 2]
```diff
@@ -6,14 +6,14 @@ <h4 class="modal-title" jhiTranslate="entity.delete.title">Confirm delete operat
...
+            <fa-icon [icon]="'ban'"></fa-icon>&nbsp;<span jhiTranslate="entity.action.cancel"></span>
         </button>
-        <button id="jhi-confirm-delete-theory" type="submit" class="btn btn-danger">
-            <fa-icon [icon]="'times'"></fa-icon>&nbsp;<span jhiTranslate="entity.action.delete">Delete</span>
```
Comment (xxMikhailx) [inline src/main/webapp/app/entities/theory/theory-delete-dialog.component.html:16]: Don't delete English words in tags

File: src/main/webapp/i18n/ru/theory.json [hunk 3]
```diff
@@ -2,24 +2,24 @@
   "learnportalApp": {
     "theory": {
       "home": {
-        "title": "Theories",
-        "createLabel": "Создать новый Theory",
-        "createOrEditLabel": "Создать или отредактировать Theory",
-        "notFound": "No Theories found"
+        "title": "Теории",
```
Comment (xxMikhailx) [inline src/main/webapp/i18n/ru/theory.json]: Change it to `Теория`

File: src/main/webapp/i18n/ru/theory.json [hunk 4]
```diff
@@ -2,24 +2,24 @@
...
+        "createLabel": "Создать новую теорию",
+        "createOrEditLabel": "Создать или отредактировать теорию",
+        "notFound": "Теории не найдены"
```
Comment (xxMikhailx) [inline src/main/webapp/i18n/ru/theory.json]: Change it to `Теория не найдена`

**Review by xxMikhailx — APPROVED** [review]

File: src/main/webapp/app/entities/theory/theory-delete-dialog.component.html (hunk 1 above)
Comment (IvanYarochkin) [reply src/main/webapp/app/entities/theory/theory-delete-dialog.component.html:13]: Done.

File: src/main/webapp/app/entities/theory/theory-delete-dialog.component.html (hunk 2 above)
Comment (IvanYarochkin) [reply src/main/webapp/app/entities/theory/theory-delete-dialog.component.html:16]: Done.

File: src/main/webapp/i18n/ru/theory.json (hunk 3 above)
Comment (IvanYarochkin) [reply src/main/webapp/i18n/ru/theory.json]: Done.

File: src/main/webapp/i18n/ru/theory.json (hunk 4 above)
Comment (IvanYarochkin) [reply src/main/webapp/i18n/ru/theory.json]: Done.

**Pull request merged**